drivers/storage/s3fs/utils/utils.go

```diff
@@ -0,0 +1,25 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_s3fs
+
+package utils
+
+import (
+	"os"
+
+	gofig "github.com/akutz/gofig/types"
+
+	"github.com/codedellemc/libstorage/api/types"
+	"github.com/codedellemc/libstorage/drivers/storage/s3fs"
+)
+
+// InstanceID returns the instance ID for the local host.
+func InstanceID(
+	ctx types.Context, config gofig.Config) (*types.InstanceID, error) {
+
+	var hostName string
+	if config == nil {
+		hostName = config.GetString(s3fs.ConfigS3FSHostName)
+	} else {
+		hostName, _ = os.Hostname()
```

Comment:
HI @codenrhoden,

Yes, the S3FS driver is not at all required to be used on EC2 instances. I'll add that to the documentation if you think it is important to call it out. I figured anyone wanting to use S3 would understand that, but there's no harm in calling it out.

The hostname and instance ID are not used by the storage driver except for logging. Even there it could be useful for them to be unique I suppose.